datastore-v1-proto-client/src/main/java/com/google/datastore/v1/client/QuerySplitterImpl.java

```diff
@@ -63,7 +66,23 @@ private QuerySplitterImpl() {
   public List<Query> getSplits(
       Query query, PartitionId partition, int numSplits, Datastore datastore)
       throws DatastoreException, IllegalArgumentException {
+    return getSplitsInternal(query, partition, numSplits, datastore, null);
+  }
 
+  @Override
+  public List<Query> getSplits(
```

Comment:
please add `@BetaApi` in appropriate places

Reply:
Done